tests/ops/op_math/test_controlled_decompositions.py

```diff
@@ -92,9 +92,6 @@ def test_invalid_op_error(self):
         qml.RY(0.123, wires=0),
         qml.RZ(0.123, wires=0),
         qml.Rot(0.123, 0.456, 0.789, wires=0),
-    ]
-
-    special_unitary_ops = [
```

Comment:
I think @albi3ro and my comment were to move 
```python
qml.Hadamard(0),
qml.PauliZ(0),
qml.S(0),
qml.PhaseShift(1.5, wires=0),
```     
to the `general_unitary_ops` list, since they are not SU(2) ops.

Reply:
Nice catch! :star_struck: Fixed!